librz/bin/format/mach0/mach064_is_kernelcache.c

```diff
@@ -1,4 +1,8 @@
+// SPDX-FileCopyrightText: 2019-2020 Francesco Tamagni <mrmacete@protonmail.ch>
+// SPDX-FileCopyrightText: 2019 pancake <pancake@nopcode.org>
+// SPDX-FileCopyrightText: 2029 ret2libc <sirmy15@gmail.com>
```

Comment:
???? :D what is this? remove, i think i didn't do anything on this... but maybe i will do it in 2029

Reply:
Sorry, time machine leaks fuel again.